swan-lake/integration-tutorials/sending-a-message-to-a-service.md

```diff
@@ -39,20 +39,9 @@ Follow the instructions given in this section to develop the service.
 
     ![Open diagram view](/learn/images/integration-tutorials/sending-a-message-to-a-service/open_diagram_view.gif)
 
-3. Define two [configurable variables](https://ballerina.io/learn/by-example/#configurability) for the port on which the listener should listen and the URL of the backend service.
+3. Define a record type named `Doctor` corresponding to the payload from the backend service.
 
-    ![Define configurable variables](/learn/images/integration-tutorials/sending-a-message-to-a-service/define_configurable_variables.gif)
-
-    The corresponding source code that is generated is as follows.
-
-    ```ballerina
-    configurable int port = 8290;
-    configurable string healthcareBackend = "http://localhost:9090/healthcare";
-    ```
-
-4. Generate a record type corresponding to the payload from the backend service using the ["Paste JSON as record"](https://wso2.com/ballerina/vscode/docs/references/convert-json-to-records/#via-the-command-palette) VS Code command by providing a sample of the expected JSON payload.
-
-    The payload from the backend service will be an array of JSON objects, where each JSON object will be similar to the following.
+    The payload from the backend service will be an array of JSON objects, where each JSON object will be similar to the following and can be modeled as `Doctor` record type.
```

Comment:
I don't think this is what @sameerajayasoma meant in Suggestion 2. We can describe how using records help with data validation, access, etc.

Reply:
What I have understood here is that we better show _the reason for defining a record before other steps_ - Payload is a list of `Doctor`

Reply:
IMO, the existing version already conveyed what we have now. 

> Generate a record type corresponding to the payload from the backend service using the ["Paste JSON as record"](https://wso2.com/ballerina/vscode/docs/references/convert-json-to-records/#via-the-command-palette) VS Code command by providing a sample of the expected JSON payload.
> 
> The payload from the backend service will be an array of JSON objects, where each JSON object will be similar to the following.

It will be good to include why. We can say that we can work with the JSON payload directly, but using records have these benefits...

Reply:
Updated. Please check

Reply:
![image](https://github.com/ballerina-platform/ballerina-dev-website/assets/25485997/754364f6-c916-4201-81d7-6d89d4b958d2)